core/templates/pages/learner-dashboard-page/community-lessons-tab.component.ts

```diff
@@ -59,6 +59,8 @@ export class CommunityLessonsTabComponent {
   @Input() collectionPlaylist!: CollectionSummary[];
   @Input() subscriptionsList!: ProfileSummary[];
   @Input() completedToIncompleteCollections!: string[];
+  @Input() redesignFeatureFlag!: boolean;
```

Comment:
Currently, we have 3 different projects regarding redesigning i.e., Topic viewer, Practice session, and learner dashboard. So can you please make this more specific to the project context?

Reply:
Done